src/fabric/src/fabric2_server.erl

```diff
@@ -325,6 +349,12 @@ apply_tx_option(Db, Option, Val, binary) ->
         true ->
             set_option(Db, Option, BinVal);
         false ->
+            ?LOG_ERROR(#{
+                what => invalid_transaction_option_value,
+                option => Option,
+                value => Val,
+                details => "string transaction option must be less than 16 bytes"
```

Comment:
Would it be `less than` or `less than or equal`?

Reply:
The `case` statement checks for `< 16` so that's what I put into the log message. I know the previous log message said differently. I didn't check if a 16 byte was actually allowed by `erlfdb` or not; if it is, we should update the test in the `case` statement as well as the log message.

Reply:
Good point. I think the check is wrong. I remember going  by the other client's API docs (Python I think) https://apple.github.io/foundationdb/api-python.html#database-options and it said `up to 16 hexadecimal digits`. But we can fix it in a separate PR.

Out of curiosity, had actually disabled the check locally and set to a much higher value and nothing seems to have happened but it's better to just go by the documentation and stick to =< 16.

`fdbserver --help` also indicates the same:

```
 /usr/sbin/fdbserver --help | grep -A2 machine_id
  -i ID, --machine_id ID
                 Machine and zone identifier key (up to 16 hex characters).
                 Defaults to a random value shared by all fdbserver processes
```